csl.rnc

```diff
@@ -717,6 +718,71 @@ div {
        attribute variable { variables.standard },
        [ a:defaultValue = "long" ] attribute form { "short" | "long" }?)
 }
+[ a:documentation [ xhtml:h2 [ "cs:title Rendering Element" ] ] ]
+div {
+  ## Use to render titles if "title-main" and "title-sub" need different formatting
+  rendering-element.title =
+    element cs:title {
+      title.attributes,
+      title.formatting,
+      (title.main?, title.sub?)
+    }
+
+  title.attributes =
+
+    ## Specify if titles are split into "title-main" and "title-sub"
+    [ a:defaultValue = "never" ]
+    attribute split {
+      
+      ## Don't split titles
+      "never" 
+      | 
+      ## Matches "."", ":", "::", "!", "?" 
+      "simple" 
+      | 
+      ## Matches the values for "simple" plus "—" and ";"
+      "full" 
+      | 
+      ## Matches the values for "simple" plus ";" and ";," or ",:"
```

Comment:
Need to double check the meaning of ` ";," or ",:"` in the Chicago manual. I remember in my original proposal I had included in them in square brackets, but I don't recall the details of what I was thinking.

Reply:
Got it. `or` is part of the delimiter.

The additional Chicago delimiters are:
- `;`
- `; or,`
- `, or:`

Expressed as a regular expression, the last two are together: `[;,] or[,:]`

Reply:
I can't follow. How can `or` be part of the delimiter? Can you spell this out in an example?

Reply:
Bump